lib/tpl/dokuwiki/css/design.less

```diff
@@ -11,7 +11,7 @@
 ********************************************************************/
 
 #dokuwiki__header {
-    padding: 2em 0 1.5em;
+    padding: 2em 0 0;
```

Comment:
This makes the sidebar be too close now. I would revert this and add negative top margin to the `.pageId` instead.